daemon/daemon.go

```diff
@@ -602,6 +602,10 @@ func New() (*Daemon, error) {
 	if err != nil {
 		return nil, err
 	}
+	err = ovld.EnableHotplug()
```

Comment:
The reason it's here is a result of earlier review suggestions as a way to avoid excessive mocking in the tests (which I initially had a lot). I'll rethink it per your other comments regarding mocking and moving all this under interface manager.

Reply:
That's changed now, per other comments.